climada/util/test/test_coordinates.py

```diff
@@ -850,6 +899,150 @@ def test_bounded_refined_raster(self):
         self.assertLess(transform[5] + z.shape[0] * transform[4], bounds[1])
         self.assertGreaterEqual(transform[5] + z.shape[0] * transform[4], bounds[1] + transform[4])
 
+    def test_reproject_raster_data_shift(self):
+        """test function reproject_raster_data for geographical shift"""
+        data_in, meta_list = data_arrays_resampling_demo()
+        i = 0 # dst
+        j = 1 # src
+        data_out, meta_out = u_coord.reproject_raster_data(data_in[j],
+                                                           meta_list[j]['crs'],
+                                                           meta_list[j]['transform'],
+                                                           dst_crs=meta_list[i]['crs'],
+                                                           dst_transform=meta_list[i]['transform'],
+                                                           dst_shape=(meta_list[i]['height'],
+                                                                      meta_list[i]['width']),
+                                                           dst_resolution=None,
+                                                           dst_global_origin=None,
+                                                           dst_global_extent=None,
+                                                           dst_buffer=None,
```

Comment:
In the tests, you specify all the input parameters, even if they are unset (None). I strongly suggest to specify the input parameters in a way that reflects what a typical user would do.